Source/RapyutaSimulationPlugins/Public/Core/RRNetworkPlayerController.h

```diff
@@ -137,8 +121,35 @@ class RAPYUTASIMULATIONPLUGINS_API ARRNetworkPlayerController : public APlayerCo
      */
     void UpdateLocalClock(float InDeltaSeconds);
 
+    /**
+     * @brief Set server linear velocity to #RobotVehicleMoveComponent. Non player object can use this method to call RPC.
+     * @param InServerRobot target server-owned robot
+     * @param InClientTimeStamp
+     * @param InClientRobotPosition
+     * @param InLinearVel
+     * @note can't use rpc since this is not controlled by Paleyr. should add (Server, Reliable)
+     */
+    UFUNCTION(BlueprintCallable, Server, Reliable)
+    virtual void ServerSetLinearVel(ARRBaseRobot* InServerRobot,
+                                    float InClientTimeStamp,
+                                    const FTransform& InClientRobotTransform,
+                                    const FVector& InLinearVel);
+
+    /**
+     * @brief Set server angular velocity to #RobotVehicleMoveComponent. Non player object can use this method to call RPC.
+     * @param InServerRobot target server-owned robot
+     * @param InClientTimeStamp
+     * @param InClientRobotRotation
+     * @param InAngularVel
+     * @note can't use rpc since this is not controlled by Paleyr. should add (Server, Reliable)
```

Comment:
```suggestion
```
This seems to be an RPC now with (Server, Reliable)?

Reply:
I assume its because the robot couldnt use an rpc?
```suggestion
     * @note This method is because RRRobotBaseVehicle can't use rpc since it is not controlled/possessed by the Player. 
```